cmd/oras/pull.go

```diff
@@ -121,6 +121,7 @@ func runPull(opts pullOptions) error {
 					return nil, err
 				}
 				if len(ss) == 0 {
+					display.Print("Skipped ", display.ShortDigest(s), s.MediaType)
```

Comment:
```suggestion
					display.Print("Skipped    ", display.ShortDigest(s), s.MediaType)			
```

Reply:
And it should only be printed out in verbose mode

Reply:
As is discussed offline, there is a bug here in the `else if` block and we need to remove the `else` otherwise unnamed config content will not be skipped.

Also, @lizMSFT is working on #275 and should modify the `if` block.